internal/config/config.go

```diff
@@ -145,11 +146,15 @@ func (c *Configuration) Sanitize() error {
 // SanitizeAdmin perform some basic checks and sanitizations in the configuration.
 // Returns true if config is acceptable, error otherwise.
 func (c *Configuration) SanitizeAdmin() error {
-	if c.Admin.IssuerLogo == "" {
-		c.Admin.IssuerLogo = "http://no-logo.com"
+	if c.APIUI.ServerPort == 0 {
+		return fmt.Errorf("an API UI server port must be provided")
 	}
 
-	if c.Admin.IssuerDID == "" {
+	if c.APIUI.IssuerLogo == "" {
+		c.APIUI.IssuerLogo = "http://no-logo.com"
```

Comment:
This needs to remain empty, since the FE handles empty logos (also, let's not use arbitrary URLs)....